### ISSUE TITLE
POIs for "park" labels shown too soon

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1767]
The scene file is doing too much work here, but maybe it's no longer relevant?

```
            landuse-labels-green-areas-not-national-park:
                filter:
                    kind: [park, conservation, protected_area, nature_reserve, forest, grass]
                    any:
                    # show labels for smaller landuse areas at higher zooms
                    - { $zoom: { max: 6 }, area: true }
                    - { $zoom: [6],    area: { max: 5000000000 } }
                    - { $zoom: [7],    area: { max: 5000000000 } }
                    - { $zoom: [8],    area: { max: 1000000000 } }
                    - { $zoom: [9],    area: { max: 100000000 } }
                    - { $zoom: [10],   area: { max: 50000000 } }
                    - { $zoom: [11],   area: { max: 25000000 } }
                    - { $zoom: [12],   area: { max: 5000000 } }
                    - { $zoom: [13],   area: { max: 200000 } }
                    - { $zoom: [14],   area: { max: 50000 } }
                    - { $zoom: [15],   area: { max: 10000 } }
                    - { $zoom: [16],   area: { max: 1000 } }
                draw:
                    mapzen_icon_library:
                        visible: false
                wilderness-areas-early:
                    filter: function() { return $zoom < 9 && feature.name && (feature.name.indexOf("Wilderness") > -1 || feature.name.indexOf("BLM") > -1 || feature.protect_class == '1' || feature.protect_class == '1a' || feature.protect_class == '1b' ); }
                    draw:
                        mapzen_icon_library:
                            visible: false
                not-national-park:
                    filter: function() { return feature.name && !((feature.name.indexOf("National Park") > -1) || feature.name.indexOf("National Monument") > -1); }
                    early:
                        filter: { $zoom: { max: 8 } }
                        draw:
                            mapzen_icon_library:
                                visible: false
```

Here's the current tier logic for parks and the other kinds (which is different):

```
  - &tier2_min_zoom
    lookup:
      key: { col: way_area }
      op: '>='
      table:
        - [ 4, 1000000000 ]
        - [ 5, 1000000000 ]
        - [ 6,  150000000 ]
        - [ 7,  100000000 ]
        - [ 8,   10000000 ]
        - [ 9,    5000000 ]
        - [ 10,   1000000 ]
        - [ 11,    500000 ]
        - [ 12,    250000 ]
        - [ 13,    100000 ]
        - [ 14,     50000 ]
        - [ 15,      2000 ]
      default: 16
```

**Comment by nvkelso** [comment]
I suspect this is partially or wholly a duplicate of https://github.com/tilezen/vector-datasource/issues/1609, already addressed in the v1.7 milestone.

Let's assume the areas filters in #1609 are "correct". That leaves us with possibly:

- [ ] Don't show non-National Park, National Monument areas until zoom 8
- [ ] Don't show wilderness areas until zoom 9

**Comment by nvkelso** [comment]
And this is basically a dupe of https://github.com/tilezen/vector-datasource/issues/1608 for the forest areas. Closing.
